src/app.py

```diff
@@ -167,32 +183,34 @@ def get_hmuuid(hmuuid):
             # Add the json MIME header in response
             return Response(response=info, mimetype="application/json")
         else:
-            return Response ("Invalid request use GET to retrieve UUID information", 500)
-    except Exception as e:                                                                                                            
-        eMsg = str(e)                                                                                                                 
-        logger.error(e, exc_info=True)                                                                                                
-        return(Response("Unexpected error: " + eMsg, 500))    
-
-@app.route('/hmuuid/<hmuuid>/exists', methods=["GET"])
-@secured(groups="HuBMAP-read")
-def is_hmuuid(hmuuid):
+            return Response("Invalid request use GET to retrieve UUID information", 500)
+    except Exception as e:
+        eMsg = str(e)
+        logger.error(e, exc_info=True)
+        return (Response("Unexpected error: " + eMsg, 500))
+
+
+@app.route('/' + app_uuid + '/<uuid>/exists', methods=["GET"])
+@secured(groups=group_name + "-read")
```

Comment:
@shirey @maxsibilla with the use of AWS API Gateway, do we still need this `@secured` decorator?

Reply:
@yuanzhou Leave the secured decorators for now.  Once we've got the new gateway stuff 100% in place we can remove most (all?) of these.